sdk/eventhub/Azure.Messaging.EventHubs.Processor/README.md

```diff
@@ -237,4 +237,4 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 
 Please see our [contributing guide](./../Azure.Messaging.EventHubs/CONTRIBUTING.md) for more information.
   
-![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Fsdk%2Feventhub%2FAzure.Messaging.EventHubs.Processor%2FREADME.png)
\ No newline at end of file
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Fsdk%2Feventhub%2FAzure.Messaging.EventHubs.Processor%2FREADME.png)
```

Comment:
What is this? It doesn't seem to include anything at the end of the file.

Reply:
This is a tracking pixel that powers metrics about page views.   It's not visible, but part of our required content.